ssc/Pos/Ssc/Toss/Logic.hs

```diff
@@ -90,7 +90,10 @@ applyGenesisBlock epoch = do
 
 -- | Rollback application of 'SscPayload's in 'Toss'. First argument is
 -- 'EpochOrSlot' of oldest block which is subject to rollback.
-rollbackSsc :: (HasConfiguration, MonadToss m) => EpochOrSlot -> NewestFirst [] SscPayload -> m ()
+rollbackSsc :: (HasConfiguration, MonadToss m) =>
+    EpochOrSlot
+    -> NewestFirst [] (ComponentBlock SscPayload)
```

Comment:
Why do you pass `Component SscPayload` to this function now? I think it should still be `SscPayload`.